From 479ad3075d0d2832f5327b917c8279feaae5f6d3 Mon Sep 17 00:00:00 2001
From: Laurent Vivier <lvivier@redhat.com>
Date: Fri, 24 May 2019 12:13:01 +0100
Subject: [PATCH 1/4] redhat: fix cut'n'paste garbage in hw_compat comments
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Laurent Vivier <lvivier@redhat.com>
Message-id: <20190524121304.15530-2-lvivier@redhat.com>
Patchwork-id: 88200
O-Subject: [RHEL-AV qemu-kvm PATCH v4 1/4] redhat: fix cut'n'paste garbage in hw_compat comments
Bugzilla: 1709726
RH-Acked-by: David Gibson <dgibson@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>

Signed-off-by: Laurent Vivier <lvivier@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/core/machine.c | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index fd1594d..e808151 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -25,12 +25,12 @@
 #include "hw/mem/nvdimm.h"
 
 /* Mostly like hw_compat_2_1 but:
- *  *    we don't need virtio-scsi-pci since 7.0 already had that on
- *   *
- *    * RH: Note, qemu-extended-regs should have been enabled in the 7.1
- *     * machine type, but was accidentally turned off in 7.2 onwards.
- *      *
- *       */
+ *    we don't need virtio-scsi-pci since 7.0 already had that on
+ *
+ * RH: Note, qemu-extended-regs should have been enabled in the 7.1
+ * machine type, but was accidentally turned off in 7.2 onwards.
+ *
+ */
 GlobalProperty hw_compat_rhel_7_1[] = {
         { /* COMPAT_RHEL7.1 */
             .driver   = "intel-hda-generic",
@@ -81,8 +81,8 @@ GlobalProperty hw_compat_rhel_7_1[] = {
 const size_t hw_compat_rhel_7_1_len = G_N_ELEMENTS(hw_compat_rhel_7_1);
 
 /* Mostly like hw_compat_2_4 + 2_3 but:
- *  *  we don't need "any_layout" as it has been backported to 7.2
- *   */
+ *  we don't need "any_layout" as it has been backported to 7.2
+ */
 
 GlobalProperty hw_compat_rhel_7_2[] = {
         {
@@ -144,9 +144,9 @@ GlobalProperty hw_compat_rhel_7_2[] = {
 const size_t hw_compat_rhel_7_2_len = G_N_ELEMENTS(hw_compat_rhel_7_2);
 
 /* Mostly like HW_COMPAT_2_6 + HW_COMPAT_2_7 + HW_COMPAT_2_8 except
- *  * disable-modern, disable-legacy, page-per-vq have already been
- *   * backported to RHEL7.3
- *    */
+ * disable-modern, disable-legacy, page-per-vq have already been
+ * backported to RHEL7.3
+ */
 GlobalProperty hw_compat_rhel_7_3[] = {
     { /* HW_COMPAT_RHEL7_3 */
         .driver   = "virtio-mmio",
@@ -209,9 +209,9 @@ GlobalProperty hw_compat_rhel_7_3[] = {
 const size_t hw_compat_rhel_7_3_len = G_N_ELEMENTS(hw_compat_rhel_7_3);
 
 /* Mostly like hw_compat_2_9 except
- *  * x-mtu-bypass-backend, x-migrate-msix has already been
- *   * backported to RHEL7.4. shpc was already on in 7.4.
- *    */
+ * x-mtu-bypass-backend, x-migrate-msix has already been
+ * backported to RHEL7.4. shpc was already on in 7.4.
+ */
 GlobalProperty hw_compat_rhel_7_4[] = {
     { /* HW_COMPAT_RHEL7_4 */
         .driver   = "intel-iommu",
@@ -261,11 +261,11 @@ GlobalProperty hw_compat_rhel_7_5[] = {
 const size_t hw_compat_rhel_7_5_len = G_N_ELEMENTS(hw_compat_rhel_7_5);
 
 /* The same as hw_compat_3_0 + hw_compat_2_12
- *  * except that
- *   *   there's nothing in 3_0
- *    *   migration.decompress-error-check=off was in 7.5 from bz 1584139
- *     *
- *      */
+ * except that
+ *   there's nothing in 3_0
+ *   migration.decompress-error-check=off was in 7.5 from bz 1584139
+ *
+ */
 GlobalProperty hw_compat_rhel_7_6[] = {
     { /* HW_COMPAT_RHEL7_6 from HW_COMPAT_2_12 */
         .driver   = "hda-audio",
-- 
1.8.3.1

